generators/server/templates/src/test/java/package/cucumber/stepdefs/_StepDefs.java

```diff
@@ -2,13 +2,15 @@
 
 import <%=packageName%>.<%= mainClass %>;
 
-import org.springframework.boot.test.SpringApplicationContextLoader;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.web.WebAppConfiguration;
 import org.springframework.test.web.servlet.ResultActions;
 
+import org.springframework.boot.test.context.SpringBootTest;
+
 @WebAppConfiguration
-@ContextConfiguration(classes = <%= mainClass %>.class, loader = SpringApplicationContextLoader.class)
+@SpringBootTest
+@ContextConfiguration(classes = <%= mainClass %>.class)
```

Comment:
@erikkemperman This is a missing feature of Cucumber, we can keep tracking this issue: https://github.com/cucumber/cucumber-jvm/issues/1061